flexzpipe/FlexZPipe.py

```diff
@@ -25,26 +25,25 @@ class FlexZPipe(PipelineStage):
 #        ('photoz_pdfs', PhotozPDFFile),                                                   
         ('photoz_pdfs', HDFFile),]
     config_options = {
-        #"dz":   0.01,
-        #"zmin": 0.005,
-	#"zmax": 3.505,
         "chunk_rows": 1000,
         "bands": ["u","g","r","i","z","y"],
         "sigma_intrins": 0.05, #"intrinsic" assumed scatter, used in ODDS                  
         "odds_int": 0.99445, #number of sigma_intrins to integrate +/- around peak         
         # note that 1.95993 is the number of sigma you get for old "ODDS" =0.95            
-        #in old BPZ, 0.68 is 0.99445                                                       
-        #"point_estimate": "mode",  # mean, mode, or median          
+        #in old BPZ, 0.68 is 0.99445                                                           
         "has_redshift": True, #does the test file have redshift?
         #if so, read in and append to output file.
         "nz": 300, #Number of grid points that FZboost will calculate   
-        "model_picklefile": "/global/homes/c/chihway/FlexZPipe/flexcode_model_sqderr.pkl", #the pickle file containing the trained  flexzbooxt model.
+        "model_picklefile": "data/example/inputs/flexcode_model_sqderr.pkl", 
+        #the pickle file containing the trained  flexzbooxt model.
         "metacal_fluxes": False, #switch for whether or not to run metacal suffices
     }
 
     def run(self):
 
         starttime = time.time()
+
+        # not sure what the two lines below are for...
```

Comment:
At least several months ago, not setting the environment variable HDF5_USE_FILE_LOCKING="FALSE" led to errors when using h5py-parallel, though I have not checked whether any changes have rendered it unnecessary.
Similarly, the CECI_SETUP was required by older versions of ceci

Reply:
`CECI_SETUP` will be ignored in newer ceci versions.  `HDF5_USE_FILE_LOCKING` is still needed by lots of things - it's set in the shifter images, which are now the best way of running I think.

Maybe the best solution here would be to add an `env` section to ceci so you can set things like this manually.

Reply:
I've updated the comment here.